### PR TITLE
Hotfix branch

### PR DESCRIPTION
**Pull request by andreea-pasare**
#475 
fixed the following:

- Remove alll of the authority tables from the restrictions file.
- Remove code lists that do no exist the Shacl: http://publications.europa.eu/resource/authority/**new**
- URIs such as http://publications.europa.eu/resource/authority/new/certification-label-type/ and
http://publications.europa.eu/resource/authority/new/price-type that are based on our base URI will cause namespace
conflicts or ambiguity. It's essential to maintain a clear and distinct namespace to avoid such issues.

### CODE REVIEW

**Comment by cristianvasquez** [comment]
Hi, thanks. I'll go through the pull request later today

File: glossary/ePO_combined_glossary.html [hunk 1]
```diff
@@ -2553,7 +2553,7 @@ <h2 xmlns:f="http://https://github.com/costezki/model2owl#">Attributes (datatype
...
+                     <td>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [1..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..1]<br></td>
```
Comment (cristianvasquez) [inline glossary/ePO_combined_glossary.html:2556]: These are changes unrelated to the hotfix issue, they are fixing new things. 

Normally, one should keep the changes atomic

Comment (andreea-pasare) [reply glossary/ePO_combined_glossary.html:2556]: These changes are automatically generated by model2owl based on the eap binary file.

File: implementation/eCatalogue/shacl_shapes/eCatalogue_shapes.ttl [hunk 2]
```diff
@@ -238,7 +231,6 @@ cat-shape:epo-cat-Catalogue-epo-cat-comprisesCatalogueLine a sh:PropertyShape ;
     sh:sparql [ sh:select "SELECT ?this ?that WHERE { ?this <http://data.europa.eu/a4g/ontology#comprisesCatalogueLine> ?that . ?that <http://data.europa.eu/a4g/ontology#comprisesCatalogueLine> ?this .}" ] .
 
 cat-shape:epo-cat-Catalogue-epo-cat-hasActivityCode a sh:PropertyShape ;
```
Comment (cristianvasquez) [inline implementation/eCatalogue/shacl_shapes/eCatalogue_shapes.ttl:233]: This PropertyShape (cat-shape:epo-cat-Catalogue-epo-cat-hasActivityCode) got 'relaxed', it says the same as cat-shape:epo-cat-CatalogueLine-epo-cat-hasActivityCode below. 

If this is no longer part of the ontology both should go away

File: implementation/eCatalogue/shacl_shapes/eCatalogue_shapes.ttl [hunk 3]
```diff
@@ -252,7 +244,6 @@ cat-shape:epo-cat-Catalogue-epo-cat-hasSpecificPaymentArrangement a sh:PropertyS
     sh:path :hasSpecificPaymentArrangement .
 
 cat-shape:epo-cat-CatalogueLine-epo-cat-hasActivityCode a sh:PropertyShape ;
-    sh:class at-voc-new:activity-code ;
```
Comment (cristianvasquez) [inline implementation/eCatalogue/shacl_shapes/eCatalogue_shapes.ttl:255]: see above

File: implementation/eNotice/owl_ontology/eNotice.rdf [hunk 4]
```diff
@@ -49,17 +48,17 @@
...
       <rdfs:seeAlso rdf:resource="https://docs.ted.europa.eu/home/index.html"/>
-      <dct:issued rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2023-11-09</dct:issued>
+      <dct:issued rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2024-01-29</dct:issued>
       <dct:created rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2021-06-01</dct:created>
-      <owl:versionInfo>4.0.0</owl:versionInfo>
+      <owl:versionInfo>4.0.1</owl:versionInfo>
       <owl:incompatibleWith>3.1.0</owl:incompatibleWith>
-      <owl:versionIRI rdf:resource="http://data.europa.eu/a4g/ontology#not-4.0.0"/>
-      <owl:priorVersion>http://data.europa.eu/a4g/ontology#not-3.1.0</owl:priorVersion>
+      <owl:versionIRI rdf:resource="http://data.europa.eu/a4g/ontology#not-4.0.1"/>
```
Comment (cristianvasquez) [inline implementation/eNotice/owl_ontology/eNotice.rdf:60]: question, for me to know. What is the 'not' part of 
http://data.europa.eu/a4g/ontology#not-3.1.0 ? it stands for what?

File: implementation/eCatalogue/shacl_shapes/eCatalogue_shapes.ttl [hunk 5]
```diff
@@ -272,7 +263,6 @@ cat-shape:epo-cat-CatalogueLine-epo-cat-hasOrderableQuantityIncrement a sh:Prope
     sh:path :hasOrderableQuantityIncrement .
 
 cat-shape:epo-cat-CatalogueLine-epo-cat-hasOrderableUnit a sh:PropertyShape ;
-    sh:class at-voc-new:orderable-unit ;
```
Comment (cristianvasquez) [inline implementation/eCatalogue/shacl_shapes/eCatalogue_shapes.ttl:275]: The sh:PropertyShape associated with at-voc-new: should go as well.

**Review by cristianvasquez — CHANGES_REQUESTED** [review]
Thanks for the pull request!, 

For this first iteration I think any sh:PropertyShape associated to at-voc-new  needs to go as well. Leaving them partially removed can pose problems to the people that will use such ShaclShapes to validate the graph

File: implementation/eNotice/owl_ontology/eNotice.rdf (hunk 4 above)
Comment (andreea-pasare) [reply implementation/eNotice/owl_ontology/eNotice.rdf:60]: it comes from the "epo-not" prefix used for the eNotice module.

File: implementation/eCatalogue/shacl_shapes/eCatalogue_shapes.ttl (hunk 2 above)
Comment (andreea-pasare) [reply implementation/eCatalogue/shacl_shapes/eCatalogue_shapes.ttl:233]: These properties are part of the ontology and the constraints are materialised here, in the default application profile.We can discuss how to express constraints on controlled vocabularies for a future release.

**Review by cristianvasquez — APPROVED** [review]
Looks ok to me!

**Pull request merged**